ivy/functional/backends/paddle/experimental/elementwise.py

```diff
@@ -190,6 +195,11 @@ def imag(
     return paddle.imag(val)
 
 
+@with_unsupported_dtypes(
+    {"2.4.2 and below": ("int8", "int16", "int32", "int64", "uint8",
```

Comment:
We need to support all datatypes except `uint16, bfloat16`, to do this you can use ivy function in array mode, this will use paddle function but with better exception handling and support for more data types